modules/administration-guide/pages/configuring-storage-sizes.adoc

```diff
@@ -57,4 +57,12 @@ spec:
 <2> Specify a claim size on the next line or omit the next line to set the default claim size value. The specified claim size is only used when you select this storage strategy.
 <3> The claim size must be specified as a link:https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. The available quantity units include: `Ei`, `Pi`, `Ti`, `Gi`,  `Mi` and `Ki`.
 
-pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
\ No newline at end of file
+pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
+
+[IMPORTANT]
+====
+Manually modifying a PVC on the cluster that was provisioned by {prod-short} is not officially supported and may result in unexpected consequences.
+
+If you want to resize a PVC,
+modify the default workspace storage size in the `CheCluster` custom resource and restart the workspace.
```

Comment:
There's no actual need to modify the default workspace storage size in the Che-Cluster CR after modifying the PVC.

Reply:
I think after this change we should be good to merge. Sorry for being slow to follow-up on this @deerskindoll

Reply:
no worries! I'll edit the wording.

Reply:
@AObuchow I reworded your suggestion a bit. please have a look